### PR TITLE
add workbench 2.4 and add sql 3.0

### PR DESCRIPTION
**Pull request by penghuo**
Signed-off-by: Peng Huo <penghuo@gmail.com>

### Description
* add SQL workbench 2.3.1, 2.4 and 3.0
* add SQL 2.3.1 and 3.0

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Comment by penghuo** [comment]
* SQL and QueryWorkbench 3.0 ready. https://github.com/opensearch-project/sql/pull/831

**Comment by penghuo** [comment]
Not sure what is it.
```
A problem occurred configuring project ':opensearch-ml-plugin'.
> Could not resolve all artifacts for configuration ':opensearch-ml-plugin:classpath'.
```

File: manifests/3.0.0/opensearch-3.0.0.yml [hunk 1]
```diff
@@ -72,3 +72,11 @@ components:
     checks:
       - gradle:properties:version
       - gradle:dependencies:opensearch.version
+  - name: sql
+    repository: https://github.com/opensearch-project/sql.git
+    ref: main
+    platforms:
+      - linux
```
Comment (peterzhuamazon) [inline manifests/3.0.0/opensearch-3.0.0.yml:79]: Add windows please?

Comment (penghuo) [reply manifests/3.0.0/opensearch-3.0.0.yml:79]: Done.

**Review by peterzhuamazon — APPROVED** [review]

**Review by peterzhuamazon — APPROVED** [review]

**Pull request merged**